daemon/events/metrics.go

```diff
@@ -0,0 +1,21 @@
+package events
```

Comment:
What is the inferred pungency here?

Are you saying that metrics should be representative of the call tree? How do we reconcile implementation versus the exported API that metrics invariably become?